airflow/exceptions.py

```diff
@@ -395,6 +395,19 @@ class ConnectionNotUnique(AirflowException):
     """Raise when multiple values are found for the same connection ID."""
 
 
+class DownstreamTasksSkipped(BaseException):
```

Comment:
This score inherit from Exception, not BaseException (mainly nothing should use Base unless you only you need it)